packages/embed-jsx/src/lib/EmbedJsx.ts

```diff
@@ -0,0 +1,115 @@
+import { MessageEmbed, ColorResolvable } from 'discord.js';
+
+const EMBED_TYPES = ['embed', 'title', 'field', 'timestamp', 'footer', 'description', 'url', 'image'] as const;
```

Comment:
```suggestion
const EMBED_TYPES = ['embed', 'title', 'field', 'timestamp', 'footer', 'description', 'url', 'image'];
```
TypeScript can infer that it is a constant, no need for the casting

Reply:
removing this line prevents ts from inferring all the switch cases, which makes it think that `make` isn't exhaustive